plugins/modules/helm_template.py

```diff
@@ -0,0 +1,213 @@
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+
+module: helm_template
+
+short_description: Render chart templates
+
+author:
+  - Mike Graves (@gravesm)
+
+description:
+  - Render chart templates to an output directory or as text of concatenated yaml documents.
+
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  chart_ref:
+    description:
+      - Chart reference with repo prefix.
+      - Path to a packaged chart.
+      - Path to an unpacked chart directory.
+      - Absolute URL.
+    required: true
+    type: path
+  chart_repo_url:
+    description:
+      - Chart repository URL where to locate the requested chart.
+    required: false
+    type: str
+  chart_version:
+    description:
+      - Chart version to use. If this is not specified, the latest version is installed.
+    required: false
+    type: str
+  include_crds:
+    description:
+      - Include custom resource descriptions in rendered templates.
+    required: false
+    type: bool
+    default: false
+  output_dir:
+    description:
+      - Output directory where templates will be written.
+    required: false
+    type: path
+  release_values:
+    description:
+        - Values to pass to chart.
+    required: false
+    default: {}
+    aliases: [ values ]
+    type: dict
+  values_files:
+    description:
+        - Value files to pass to chart.
+        - Paths will be read from the target host's filesystem, not the host running ansible.
+        - I(values_files) option is evaluated before I(values) option if both are used.
+        - Paths are evaluated in the order the paths are specified.
+    required: false
+    default: []
+    type: list
+    elements: str
+  update_repo_cache:
+    description:
+      - Run C(helm repo update) before the operation. Can be run as part of the template generation or as a separate step.
+    default: false
+    type: bool
+'''
+
+EXAMPLES = r'''
+- name: Render templates to specified directory
+  community.kubernetes.helm_template:
+    chart_ref: stable/prometheus
+    output_dir: mycharts
+
+- name: Render templates
+  community.kubernetes.helm_template:
+    chart_ref: stable/prometheus
+  register: result
+
+- name: Write templates to file
+  copy:
+    dest: myfile.yaml
+    content: "{{ result.stdout }}"
+'''
+
+RETURN = r'''
+stdout:
+  type: str
+  description: Full C(helm) command stdout. If no I(output_dir) has been provided this will contain the rendered templates as concatenated yaml documents.
+  returned: always
+  sample: ''
+stderr:
+  type: str
+  description: Full C(helm) command stderr, in case you want to display it or examine the event log.
+  returned: always
+  sample: ''
+command:
+  type: str
+  description: Full C(helm) command run by this module, in case you want to re-run the command outside the module or debug a problem.
+  returned: always
+  sample: helm upgrade ...
```

Comment:
Is `helm upgrade` a possible `command` for this module to return or should this be a `helm template` command?